tests/test_sims.py

```diff
@@ -898,6 +898,41 @@ def setUpClass(cls):
         # define random number generator
         cls.rng = np.random.default_rng(cls.seed)
 
+    def test_ER_np(self):
+        rng = np.random.default_rng(self.seed)
+
+        p = [[1, 0.4],
+             [0.4, 0.7]]
+
+        alpha = [1000, 0.05]
+
+        n = 100 
```

Comment:
can you make this like 10 and see if it still works, trying to make the tests faster going forward

Reply:
For sure, it shouldn't cause any problem!